src/backends/custom/loader.cc

```diff
@@ -27,6 +27,7 @@
 #include "src/backends/custom/loader.h"
 
 #include <dlfcn.h>
+
```

Comment:
Remove blank line. We don't separate system and local includes...

Reply:
Removed